stix2/core.py

```diff
@@ -62,37 +59,30 @@ def __init__(self, *args, **kwargs):
         super(Bundle, self).__init__(**kwargs)
 
 
-OBJ_MAP = {
-    'attack-pattern': AttackPattern,
-    'bundle': Bundle,
-    'campaign': Campaign,
-    'course-of-action': CourseOfAction,
-    'identity': Identity,
-    'indicator': Indicator,
-    'intrusion-set': IntrusionSet,
-    'malware': Malware,
-    'marking-definition': MarkingDefinition,
-    'observed-data': ObservedData,
-    'report': Report,
-    'relationship': Relationship,
-    'threat-actor': ThreatActor,
-    'tool': Tool,
-    'sighting': Sighting,
-    'vulnerability': Vulnerability,
-}
-
-
-def parse(data, allow_custom=False):
+STIX2_OBJ_MAPS = {}
+
+
+def parse(data, allow_custom=False, version=None):
     """Deserialize a string or file-like object into a STIX object.
 
     Args:
         data (str, dict, file-like object): The STIX 2 content to be parsed.
-        allow_custom (bool): Whether to allow custom properties or not. Default: False.
+        allow_custom (bool): Whether to allow custom properties or not.
+            Default: False.
+        version (str): Which STIX2 version to use. (e.g. "2.0", "2.1"). If
+            None, use latest version.
 
     Returns:
         An instantiated Python STIX object.
 
     """
+    if not version:
+        # Use latest version
+        OBJ_MAP = STIX2_OBJ_MAPS[sorted(STIX2_OBJ_MAPS.keys())[-1]]
```

Comment:
Would it help to have a global `stix2.DEFAULT_VERSION` variable that makes it easier for code that uses python-stix2 to check what version of STIX is supported by default?